### PR TITLE
fix: #1459 #1461 Add redirect urls for SPAR and SILVA

### PR DESCRIPTION
**Pull request by MCatherine1994**
refs: #1459, #1461

### CODE REVIEW

File: infrastructure/server/oidc_clients_spar.tf [hunk 1]
```diff
@@ -43,9 +43,11 @@ resource "aws_cognito_user_pool_client" "test_spar_oidc_client" {
   callback_urls                                 = concat([
     "http://localhost:3000/",
     "https://oidcdebugggersecure-c6af30-dev.apps.gold.devops.gov.bc.ca/",
-    "https://nr-spar-test-frontend.apps.silver.devops.gov.bc.ca/"
+    "https://nr-spar-test-frontend.apps.silver.devops.gov.bc.ca/",
```
Comment (ianliuwk1019) [inline infrastructure/server/oidc_clients_spar.tf:46]: Wondering do they need to remove the old one?

Comment (ianliuwk1019) [reply infrastructure/server/oidc_clients_spar.tf:46]: Never mind, seems the ticket mentioned to keep the old one for now.

**Review by ianliuwk1019 — APPROVED** [review]

File: infrastructure/server/oidc_clients_spar.tf (hunk 1 above)
Comment (MCatherine1994) [reply infrastructure/server/oidc_clients_spar.tf:46]: last time when they sent me the message they want to keep the old ones, cause they haven't finish transferring to the new urls. But I'll ask them again next week just to make sure they still need the old ones.

**Comment by MCatherine1994** [comment]
confirmed with Craig, they want to keep the existing urls as well for a couple more weeks before their prod release

**Pull request merged**